### DISCUSSION TITLE
Duda ENTREGAR-PEDIDOS

### DISCUSSION THREAD

**Discussion by roahoki** [IIC2133-PUC/2024-1#34] [Tarea 0] [answered]
holaa, me quedó una duda con el item 2-2 de entregar pedidos. Se espera que al ocurrir este evento las naves desde 0 hasta N-1 entreguen su pedido más prioritario, es decir, solamente 1? o se espera que si, por ejemplo, la nave 2 tiene que el pedido más prioritario en el planeta 1 y todos los pedidos restantes que tienen son en ese planeta, entonces debe entregar todos sus pedidos de ese planeta.

En resumen, quiero saber si cada nave debe entregar el pedido prioritario y nada más o debe entregar el más prioritario y todos los otros pedidos que tenga en ese planeta

**Comment by PabloNavarroY** [comment, marked as answer]
Lo segundo. #21

**Comment by roahoki** [comment]
jajaj si, lo leí mejor. Redacción de ese ítem 4/10
